### PR TITLE
feat(notifications) trigger iframe api event when a notification occurs

### PR DESCRIPTION
**Pull request by quitrk**

### CODE REVIEW

File: react/features/notifications/actions.ts [hunk 1]
```diff
@@ -116,12 +116,20 @@ export function showNotification(props: INotificationProps = {}, type?: string)
...
-                || disabledNotifications.includes(props.titleKey ?? ''))
+            && !(disabledNotifications.includes(descriptionKey ?? '')
+                || disabledNotifications.includes(titleKey ?? ''))
             && (!notifications
-                || notifications.includes(props.descriptionKey ?? '')
-                || notifications.includes(props.titleKey ?? ''));
+                || notifications.includes(descriptionKey ?? '')
+                || notifications.includes(titleKey ?? ''));
+
+        if (typeof APP !== 'undefined') {
```
Comment (horymury) [inline react/features/notifications/actions.ts:128]: shouldn't it be sent only when shouldDisplay is true? Or add shouldDisplay on the API notification payload?

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/notifications/actions.ts (hunk 1 above)
Comment (quitrk) [reply react/features/notifications/actions.ts:128]: no, we always want the event to be triggered, leaving it up to the consumer to handle it whichever way they want

**Review by horymury — DISMISSED** [review]

**Review by saghul — DISMISSED** [review]

**Comment by quitrk** [comment]
Jenkins please test this please

**Comment by saghul** [comment]
Jenkins please test this please.

File: react/features/notifications/actions.ts [hunk 2]
```diff
@@ -116,12 +116,20 @@ export function showNotification(props: INotificationProps = {}, type?: string)
...
+            // when we are redirecting the library should handle any
+            // unload and clean of the connection.
```
Comment (horymury) [inline react/features/notifications/actions.ts:130]: don't think the comment applies to this API command :D

**Review by horymury — CHANGES_REQUESTED** [review]

**Review by horymury — DISMISSED** [review]

**Pull request merged**